common/web/types/src/main.ts

```diff
@@ -50,7 +50,7 @@ export { KeymanDeveloperProject, KeymanDeveloperProjectFile, KeymanDeveloperProj
 export * as KpsFile from './package/kps-file.js';
 export * as KmpJsonFile from './package/kmp-json-file.js';
 
-export * as util from './util/util.js';
+export { Uni_IsSurrogate1, Uni_IsSurrogate2 } from './util/util.js';
```

Comment:
```suggestion
export { Uni_IsSurrogate1, Uni_IsSurrogate2 } from './util/util.js';
export * as util from './util/util.js';
```

No reason we can't keep the original export point in place for Developer usages.